homeassistant/components/notify/facebook.py

```diff
@@ -55,20 +58,27 @@ def send_message(self, message="", **kwargs):
             _LOGGER.error("At least 1 target is required")
             return
 
-        for target in targets:
-            # If the target starts with a "+", we suppose it's a phone number,
-            # otherwise it's a user id.
-            if target.startswith('+'):
-                recipient = {"phone_number": target}
-            else:
-                recipient = {"id": target}
-
-            body = {
-                "recipient": recipient,
-                "message": body_message
+        # broadcast message
+        if targets[0] == 'BROADCAST':
```

Comment:
Why do you assume the list of targets has just one entry? May be all targets should be handled here, too?

Reply:
Hi Sebastian, thank you very much for looking at my PR. 
This is by nature of the functionality of "BROADCAST", only ONE target is required for broadcast to work. And the name of that target is "BROADCAST". Only ONE target will be considered and thus it has to be in the targets[0] location. We will ignore anything else if it is present. This is under if/else statement basically and all the other cases where we have multiple targets are handled by the original code.

Reply:
Publishing a broadcast and address some people at once shouldn't be possible?

Reply:
yes, indeed, not possible. Once you put the keyword "BROADCAST" as a first entry, any specific targets are implicitly ignored. So the code will NOT break even if people do not remove specific targets.. In my view the code is very robust the way it is implemented, as it will ignore "small" misconfigurations rather than bomb out with an error.

Reply:
Alright. Thanks for the explanation!

Reply:
I have amended the code and now also check for length of target to be 1. If not, I'll pass it to the  original code to deal with it in the same way as before. So now, we will only do BROADCAST if they correctly configure it for broadcast. I think once it builds , I hopefully satisfied all requirements. :-) let me know once it runs thru all the tests.